### ISSUE TITLE
Batch normalization for  Bidirectional RNN ?tensorflow support it?

### ISSUE THREAD

**Issue by guoying1030** [tensorflow/tensorflow#7432]
Batch normalization for  Bidirectional RNN ?tensorflow support it?

**Comment by aselle** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
